docs/en/other.md

```diff
@@ -4,6 +4,16 @@ pageClass: routes
 
 # Uncategorized
 
+## 591 Rental house
+
+<Route author="Yukaii" example="/591/query/order=posttime&orderType=desc" path="/591/query/:query?" :paramsDesc="['Query Parameters']"/>
```

Comment:
It is suggested to add `tw` in `path` to avoid confusion between `rent.591.com.hk` and `rent.591.com.tw`

Reply:
fixed in bb9c3b6